src/source/Sctp/Sctp.c

```diff
@@ -213,11 +212,52 @@ STATUS sctpSessionWriteDcep(PSctpSession pSctpSession, UINT32 streamId, PCHAR pC
      CHK(pSctpSession != NULL && pChannelName != NULL, STATUS_NULL_ARG);
      CHK((pPacket = (PBYTE) MEMCALLOC(1, pPacketSize)) != NULL, STATUS_NOT_ENOUGH_MEMORY);
 
-     pPacket[0] = DCEP_DATA_CHANNEL_OPEN;
+     /* Setting the fields of DATA_CHANNEL_OPEN message */
+
+     pPacket[0] = DCEP_DATA_CHANNEL_OPEN; // message type
+
+     // Set Channel type based on supplied parameters
      pPacket[1] = DCEP_DATA_CHANNEL_RELIABLE;
+
+     // Set channel type and reliability parameters based on input
+     // SCTP allows fine tuning the channel robustness:
+     // 1. Ordering: The data packets can be sent out in an ordered/unordered fashion
+     // 2. Reliability: This determines how the retransmission of packets is handled.
+     // There are 2 parameters that can be fine tuned to achieve this:
+     //     a. Number of retransmits
+     //     b. Packet lifetime
+     // Default values for the parameters is 0. This falls back to reliable channel
+     if(pRtcDataChannelInit != NULL) {
+         switch(pRtcDataChannelInit->ordered) {
```

Comment:
use if/else with boolean type not switch

Reply:
Changed to if/else.